packages/token-bridge-sdk/src/hooks/arbTokenBridge.types.ts

```diff
@@ -47,25 +47,116 @@ export enum AssetType {
   ETH = 'ETH'
 }
 
-export type TransactionLifecycle<Tx, TxReceipt> = Partial<{
-  onTxSubmit: (tx: Tx) => void
-  onTxConfirm: (txReceipt: TxReceipt) => void
-}>
+export enum TokenSymbol {
+  ETH = 'ETH'
+}
```

Comment:
Should we keep an enum around if it's only gonna have the one member?

Reply:
I'd prefer to use an enum `assetName: TokenSymbol.ETH` over `assetName: 'ETH'` multiple times :/